assets/src/blocks/HubspotForm/HubspotFormEditor.js

```diff
@@ -0,0 +1,260 @@
+import {
+  PanelBody,
+  Button,
+} from '@wordpress/components';
+import {
+  InspectorControls,
+  MediaUpload,
+  MediaUploadCheck,
+} from '@wordpress/block-editor';
+import {
+  Fragment,
+  useCallback,
+  useState,
+  useEffect,
+  useMemo,
+  useContext,
+} from '@wordpress/element';
+import { HubspotFormContext, HubspotFormProvider } from './HubspotFormContext';
+import { URLInput } from '../../components/URLInput/URLInput';
+
+const { __ } = wp.i18n;
+const { RichText } = wp.blockEditor;
+
+const Edit = ({
+  attributes: {
+    hubspot_portal,
+    block_button_link,
+  },
+  setAttributes,
+}) => {
+
+  const onChangeHandler = (evt) => {
+    evt.preventDefault();
+    setAttributes({ hubspot_portal: evt.currentTarget.value });
+  }
+
+  return useMemo(() => (
```

Comment:
Why do we have to use `useMemo` here?

Reply:
I see it's used pretty much be default. While `useMemo` **can** in some cases be used to address a performance issue if there's a particularly expensive calculation, in most cases it actually reduces performance. Please check  https://medium.com/swlh/should-you-use-usememo-in-react-a-benchmarked-analysis-159faf6609b7 to get an idea of the complexity needed in the calculation for `useMemo` to actually improve performance.